src/helpers/buildTransactionsSectionsSelector.tsx

```diff
@@ -61,7 +61,7 @@ const addContactInfo =
     contact: Contact | null;
   } => {
     const { from, to, status } = txn;
-    const isSent = status === TransactionStatus.sent;
+    const isSent = status === TransactionStatus.confirmed;
```

Comment:
this has been broken for a while it seems - but in order to restore it, we should follow the true definition of `isSent` and check TransactionType is `send` and  TransactionStatus is `confirmed`